Resources/Public/Css/TableOfContents.css

```
@@ -0,0 +1,148 @@
+.typo3-styleguide-toc__card-icon img,
+.typo3-styleguide-toc__card-icon svg {
+    width: 48px;
+    height: 48px;
+    object-fit: contain;
+}
+
+.typo3-styleguide-toc__card-title {
+    font-family: monospace, sans-serif;
+    font-size: 0.95rem;
+    text-align: center;
+    word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)